README.md

```diff
@@ -1,17 +1,15 @@
 # Logo File Icons
 
-Add some nice logo icons to your files in Atom. Logos were taken from http://svgporn.com/
+Add some nice logo icons to your files in Atom. Logos were taken from [SVGPORN](http://svgporn.com)
 
 All logos are the property of their respective owners.
 
 ## Preview
 
 ### Light theme
-![Preview light](https://raw.githubusercontent.com/manuelbieh/logo-file-icons/master/logo-file-icons.png)
+![Preview-light](https://cloud.githubusercontent.com/assets/16760489/21432324/931e4c9a-c89d-11e6-9309-6707ecdaddb9.png)
```

Comment:
I read some README file like [setings-view](https://github.com/atom/settings-view), [markdown-preview](https://github.com/atom/markdown-preview) so I think it shoulds change.